airflow/jobs/scheduler_job.py

```diff
@@ -1159,17 +1159,24 @@ def _change_state_for_tis_without_dagrun(
             tis_to_change: List[TI] = query.with_for_update().all()
             for ti in tis_to_change:
                 ti.set_state(new_state, session=session)
+                ti.duration = 0
```

Comment:
Shouldn't this be in ti.duration?

Reply:
in `ti.set_state` right? fixed in https://github.com/apache/airflow/pull/11358/commits/3b0d977d413b38e6b394287a6cbc85a6bc25b414